x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/action/ResponseValueUtils.java

```diff
@@ -38,7 +38,9 @@
 import java.util.List;
 import java.util.Map;
 
-import static org.elasticsearch.xpack.ql.util.DateUtils.UTC_DATE_TIME_FORMATTER;
+import static org.elasticsearch.xpack.esql.type.EsqlDataTypeConverter.ESQL_DEFAULT_DATE_TIME_FORMATTER;
```

Comment:
No need to add the `ESQL_` prefix, it's already qualified through the class name (EsqlDataTypeConverter).